src/api/SOOSAnalysisApiClient.ts

```diff
@@ -29,12 +36,12 @@ interface ICreateScanRequest {
   branchUri: string | null;
   integrationType: string;
   operatingEnvironment: string;
-  integrationName: string | null;
+  integrationName?: IntegrationName | null;
```

Comment:
can we make this always required?
```suggestion
  integrationName: IntegrationName;
```